spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/StandardizationParquetSuite.scala

```diff
@@ -363,41 +327,42 @@ class StandardizationParquetSuite extends FixtureAnyFunSuite with SparkTestBase
     )
     val schema = StructType(seq)
     // stableHashId will always yield the same ids
-    val destDF = StandardizationInterpreter.standardize(sourceDF, schema, cmd.rawFormat, recordIdGenerationStrategy = IdType.StableHashId)
+    val actualDf = StandardizationInterpreter.standardize(sourceDF, schema, cmd.rawFormat, recordIdGenerationStrategy = IdType.StableHashId)
+
+    val expectedData = Seq(
+      Row(1L, Array("A", "B"), Row(false), Array(), 1950798873),
+      Row(2L, Array("C"), Row(true), Array(), -988631025)
+    )
+    val expectedDF = expectedData.toDfWithSchema(actualDf.schema) // checking the just data, not the schema here
```

Comment:
```suggestion
    val expectedDF = expectedData.toDfWithSchema(actualDf.schema) // checking just the data, not the schema here
```
I think it's copy pasted on multiple places.

Reply:
Yes it is. Thanks for finding this, I was already blind to it. Fixed.